.env.development

```diff
@@ -0,0 +1,4 @@
+PLACEMENTS_HOST=placements.localhost
```

Comment:
Do we need to include the .env* files in this PR?

Reply:
Fixed this by adding one file .env.*